src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObject = {};
+
+  const splited = sourceString.split(';');
```

Comment:
Instead of checking ``if (split.trim())`` in each iteration, you can trim whitespace at the very beginning and then filter out empty lines.

```suggestion
  const splited = sourceString
    .split(';')
    .map((declaration) => declaration.trim())
    .filter((declaration) => declaration);
```